src/graphics.rs

```diff
@@ -1002,13 +1002,17 @@ impl From<BlendFactor> for GLenum {
 
 #[derive(Debug, PartialEq, Clone, Copy)]
 pub struct PipelineParams {
-    pub cull_face: CullFace,
-    pub front_face_order: FrontFaceOrder,
+    pub cull_face: CullFace, // TODO: UNUSED, should call glCullFace
+    pub front_face_order: FrontFaceOrder, // TODO: UNUSED, should call glFrontFace
     pub depth_test: Comparison,
     pub depth_write: bool,
     pub depth_write_offset: Option<(f32, f32)>,
     pub color_blend: Option<BlendState>,
     pub color_write: ColorMask,
+    // TODO: support:
+    // glStencilMaskSeparate(...);
+    // glStencilFuncSeparate(...);
+    // glStencilOpSeparate(...);
```

Comment:
@not-fl3 do you have any preferences or suggestions on how to structure the PipelineParams to support the above 3 functions? I'm still thinking about it.

Reply:
This is how it looks in sokol_gfx, just for reference:

```C
typedef struct sg_pipeline_desc {
..
    sg_depth_stencil_state depth_stencil;
..
}
```

```C
typedef struct sg_depth_stencil_state {
    sg_stencil_state stencil_front;
    sg_stencil_state stencil_back;
    sg_compare_func depth_compare_func;
    bool depth_write_enabled;
    bool stencil_enabled;
    uint8_t stencil_read_mask;
    uint8_t stencil_write_mask;
    uint8_t stencil_ref;
} sg_depth_stencil_state;
```

Maybe we can do similar to `color_blend`? 
Something like
```rust
...
stencil: Option<StencilState>
...

```rust
struct BlendState {
    front: Foo,
    back: Foo,
    read_mask: Mask,
    write_mask: Mask,
    ref: Foo
}
```

Not sure, just first idea :)

Reply:
Created new PR with just the options.